src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.Keys.cs

```diff
@@ -221,6 +239,97 @@ public static void ExportDecapsulationKey_OnlyHasEncapsulationKey()
                 new byte[vector.Algorithm.DecapsulationKeySizeInBytes]));
         }
 
+        [ConditionalTheory(typeof(MLKem), nameof(MLKem.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public static void SubjectPublicKeyInfo_MLKem512_Ietf(bool tryExport)
```

Comment:
Bonus points if you rename all of the `bool tryExport` callers to `bool useTryExport`.  Which, silly me, I thought was implied with renaming it on the assert helper :smile:

Reply:
> Which, silly me, I thought was implied with renaming it on the assert helper 😄

Ah, no, that's a reasonable assumption. That's a miss on my part.